### PR TITLE
Update docs: do not use deprecated create method

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

**Review by julienfalque — APPROVED** [review]
Thanks :)

File: .php_cs.dist [hunk 1]
```diff
@@ -19,7 +19,8 @@ $finder = PhpCsFixer\Finder::create()
     ])
 ;
 
-$config = PhpCsFixer\Config::create()
```
Comment (kubawerlos) [inline .php_cs.dist:22]: Why can't we go with `$config = (new PhpCsFixer\Config())`?

Comment (keradus) [reply .php_cs.dist:22]: not available in each supported PHP version

Comment (kubawerlos) [reply .php_cs.dist:22]: @keradus are you sure? https://3v4l.org/OItKe

Comment (keradus) [reply .php_cs.dist:22]: ha, so it started to work since 5.4? i though that only 7.0.
nice ;)

**Pull request merged**